### ISSUE TITLE
diffoscope on High Sierra: ModuleNotFoundError: No module named 'pkg_resources'

### ISSUE THREAD

**Issue by aviflax** [Homebrew/homebrew-core#33645]
- [x] are reporting a bug others will be able to reproduce and not asking a question. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh
- [x] have a problem with `brew install` (or `upgrade`, `reinstall`) a single, official formula (not cask)? If it's a general `brew` problem please file this issue at Homebrew/brew: https://github.com/Homebrew/brew/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/homebrew-cask/issues/new/choose. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [x] ran `brew update` and can still reproduce the problem?
- [x] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [x] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link? **I ran this there was no output.**

- [x] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

<details>
<summary><b><code>brew config</code></b></summary>
<pre><code>
HOMEBREW_VERSION: 1.8.1
ORIGIN: https://github.com/Homebrew/brew
HEAD: 422257067cc46d86138baa0e8e453348c2829cee
Last commit: 2 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 86b0dfafbc69f3b7cc4a04997f1689196e569b4d
Core tap last commit: 3 hours ago
HOMEBREW_PREFIX: /usr/local
CPU: quad-core 64-bit haswell
Homebrew Ruby: 2.3.7 => /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/bin/ruby
Clang: 10.0 build 1000
Git: 2.19.1 => /usr/local/bin/git
Curl: 7.54.0 => /usr/bin/curl
Java: 1.8.0_171, 1.8.0_151
macOS: 10.13.6-x86_64
CLT: 10.1.0.0.1.1539992718
Xcode: 10.1
</code></pre>
</details>

<details>
<summary><b><code>brew doctor</code></b></summary>
<pre><code>
Your system is ready to brew.
</code></pre>
</details>

<br>

To help us debug your issue please explain:

- What you were trying to do (and why)

Install and use `diffoscope`

- What happened (include command output)

```sh
$ diffoscope
Traceback (most recent call last):
  File "/usr/local/Cellar/diffoscope/104/libexec/bin/diffoscope", line 6, in <module>
    from pkg_resources import load_entry_point
ModuleNotFoundError: No module named 'pkg_resources'
```

- What you expected to happen

I’d like diffoscope to run successfully.

- Step-by-step reproduction instructions (by running `brew install` commands)

Basically I just ran `brew install diffoscope` solved some misc general issues with my system’s config, then ran the command again and it succeeded.

The misc general issues I solved were just because I hadn’t used brew for a good while, I think. Permissions errors, Xcode licensing errors, command-line-tools not being installed, I think that’s it.

**Comment by aviflax** [comment]
Well, that’s disconcerting.

**Comment by Moisan** [comment]
`diffoscope` was updated recently, can you still reproduce the issue?

**Comment by aviflax** [comment]
Works for me now. Thank you!
